admin-ui/plugins/auth-server/components/AuthN/Basic.js

```diff
@@ -30,23 +28,12 @@ function AuthNListPage({ isBuiltIn = false }) {
   const themeColors = getThemeColor(selectedTheme)
   const bgThemeColor = { background: themeColors.background }
 
-  const [list, setList] = useState({
-    ldap: [],
-    scripts: [],
-  })
-
   const authN = useSelector((state) => state.authNReducer.acrs)
-  const ldap = useSelector((state) => state.ldapReducer.ldap)
-  const scripts = useSelector((state) => state.customScriptReducer.items)
-  const scriptsLoading = useSelector((state) => state.customScriptReducer.loading)
-  const loading = useSelector((state) => state.ldapReducer.loading)
   const acrs = useSelector((state) => state.acrReducer.acrReponse)
-  const customScriptloading = useSelector((state) => state.customScriptReducer.loading)
   const { permissions: cedarPermissions } = useSelector((state) => state.cedarPermissions)
 
-  SetTitle(t('titles.authn'))
+  SetTitle('Built In')
```

Comment:
Consider using translation for page title

The page title "Built In" is hardcoded. For consistency with internationalization, consider using a translation key.



```diff
-  SetTitle('Built In')
+  SetTitle(t('pages.built_in'))
```